binding/rust/src/util.rs

```diff
@@ -84,92 +88,97 @@ fn base_library_path() -> PathBuf {
         machine if RPI_MACHINES.contains(&machine) => {
             if cfg!(target_arch = "aarch64") {
                 PathBuf::from(format!(
-                    "lib/raspberry-pi/{}-aarch64/libpv_porcupine.so",
+                    "raspberry-pi/{}-aarch64/libpv_porcupine.so",
                     &machine
                 ))
             } else {
-                PathBuf::from(format!("lib/raspberry-pi/{}/libpv_porcupine.so", &machine))
+                PathBuf::from(format!("raspberry-pi/{}/libpv_porcupine.so", &machine))
             }
         }
         machine if JETSON_MACHINES.contains(&machine) => {
-            PathBuf::from("lib/jetson/cortex-a57-aarch64/libpv_porcupine.so")
+            PathBuf::from("jetson/cortex-a57-aarch64/libpv_porcupine.so")
         }
-        "beaglebone" => PathBuf::from("lib/beaglebone/libpv_porcupine.so"),
+        "beaglebone" => PathBuf::from("beaglebone/libpv_porcupine.so"),
         _ => {
             warn!("WARNING: Please be advised that this device is not officially supported by Picovoice.\nFalling back to the armv6-based (Raspberry Pi Zero) library. This is not tested nor optimal.\nFor the model, use Raspberry Pi's models");
-            PathBuf::from("lib/raspberry-pi/arm11/libpv_porcupine.so")
+            PathBuf::from("raspberry-pi/arm11/libpv_porcupine.so")
         }
     };
 }
 
 pub fn pv_library_path() -> PathBuf {
     let mut path = PathBuf::from(file!());
     path.pop(); // file! macro includes filename
-    path.push("../");
+    path.push(DEFAULT_RELATIVE_LIBRARY_DIR);
     path.push(base_library_path());
     return path;
 }
 
 pub fn pv_model_path() -> PathBuf {
     let mut path = PathBuf::from(file!());
     path.pop(); // file! macro includes filename
-    path.push("../");
-    path.push("lib/common/porcupine_params.pv");
+    path.push(DEFAULT_RELATIVE_MODEL_PATH);
     return path;
 }
 
 #[cfg(target_os = "macos")]
-fn keyword_path_subdir() -> PathBuf {
-    return PathBuf::from("mac");
+fn keyword_path_subdir() -> String {
```

Comment:
even though this string does represent the keyword subdir, maybe we should just call the function pv_platform because sometimes it's a directory sometimes we use it in the keyword file name to okay the platform tag.